hera_pspec/plot.py

```diff
@@ -494,3 +496,176 @@ def delay_waterfall(uvp, blpairs, spw, pol, component='real', average_blpairs=Fa
     if new_plot:
         return fig
     
+def delay_wedge(uvps, blpairs, fold=False, cosmo=True, center_line=True, horizon_lines=True, cosmo_cbar=True, suptitle=''):
+    """
+    Plot a 2D delay spectrum (or spectra) for a group of baselines.
+    
+    Parameters
+    ----------
+    uvps : UVPspec
+        List of UVPSpec objects, containing delay spectra for a set of baseline-pairs, 
+        times, polarizations, and spectral windows.
+        
+    blpairs : list of tuples
+        List of baseline-pair tuples.
+        
+    fold : bool, optional
+        Whether to fold the power spectrum in :math:`|k_\parallel|`. 
+        Default: False.
+        
+    cosmo : bool, optional
+        Whether to convert axes to cosmological units :math:`|k_\parallel|` and 
+        :math:`|k_\perpendicular|`.
+        Default: True.
+        
+    center_line : bool, optional
+        Whether to plot a dotted line at :math:`|k_\parallel|` =0.
+        Default: True.
+        
+    horizon_lines : bool, optional
+        Whether to plot dotted lines along the horizon.
+        Default: True.
+        
+    cosmo_cbar : bool, optional
+        Whether power has been converted to appropriate units through calibration.
+        Default: True.
+        
+    suptitle : string, optional
+        Suptitle for the plot.  If not provided, no suptitle will be plotted.
+        Default: ''.
+        
+    Returns
+    -------
+    f : matplotlib.pyplot.Figure
+        Matplotlib Figure instance.
+    """
+    
+    if (type(uvps) != list) and (type(uvps) != np.ndarray):
+        raise AttributeError("The uvps paramater should be a list of UVPSpec objects.")
+        
+    #Initialize axes objects
+    ncols = len(uvps)
+    f, axes = plt.subplots(
```

Comment:
only create fig and axes if `ax=None`, see other plotting functions in this script for example